### ISSUE TITLE
react-a11y-anchors fails on links that contact exactly 4 characters of text

### ISSUE THREAD

**Issue by bolatovumar** [Microsoft/tslint-microsoft-contrib#269]
I'm linting a JSX element that looks something like this:
`<a href="http://www.someurlhere.com" className="someClass">More</a>`

The rule fails. However, if add a space after "More" like so:
`<a href="http://www.someurlhere.com" className="someClass">More </a>`
It passes.

The rule states that:
> Link text should be at least 4 characters long.

**Comment by ipip2005** [comment]
I'm going to fix it since our project also hit this

**Comment by ipip2005** [comment]
https://github.com/Microsoft/tslint-microsoft-contrib/pull/284

**Comment by HamletDRC** [comment]
merging now
